frontend/src/app/corpus-info/corpus-info.component.ts

```diff
@@ -41,4 +46,12 @@ export class CorpusInfoComponent implements OnInit {
         return marked.parse(content);
     }
 
+    private sortPages(pages: CorpusDocumentationPage[]): CorpusDocumentationPage[] {
+        return _.sortBy(pages, 'index');
+    }
```

Comment:
This does not warrant a function, unless expected to do more in the future. 
The readable name is nice, but `sortBy` is intuitive enough that I prefer not bloating the component.